### ISSUE TITLE
Use the Router DHCP information - why is not possible?

### ISSUE THREAD

**Issue by titust1** [pucherot/Pi.Alert#189]
All my devices are using DHCP with reservation in the Router, so in fact they are using static addresses. The DHCP router also contains hostnames for all devices. This information is read by Pi-Hole due to the conditional forwarding, and Pi-hole is able to display the device hostnames. Why Pi-Alert is not able to do the same? 
Or at least to be able to import the device list in csv format that I exported form the router.
It's a pain to manually populate the Pi Alert when this info is already there, and it was already entered

**Comment by harr2969** [comment]
Most routers would have a different way of storing the information, meaning the approach to getting that would not be universal.  Also, most consumer routers are not open to connections that would allow getting this information easily, and that goes double when the customer is renting a router from their ISP. Just my .02c.

**Comment by iGadget** [comment]
> Most routers would have a different way of storing the information, meaning the approach to getting that would not be universal. Also, most consumer routers are not open to connections that would allow getting this information easily, and that goes double when the customer is renting a router from their ISP. Just my .02c.

Fair enough that getting this info straight from the router would perhaps be a challenge, but Pi-Hole is (in most cases) able to do it. So why not use their heavy lifting and sync the info from Pi-Hole?

**Comment by jokob-sk** [comment]
Hmmm, afaik this is already how it works. Not sure what more to get from PiHole.
